apps/mn-synt-arbeidsforhold-service/src/main/java/no/nav/registre/testnorge/mn/syntarbeidsforholdservice/provider/IdentController.java

```diff
@@ -32,6 +32,11 @@
             @RequestParam("fom") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate fom,
             @RequestParam(value = "tom", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate tom
     ) {
+        if (identService.getIdenterMedArbeidsforhold(miljo).contains(ident)) {
+            return ResponseEntity
+                    .badRequest()
+                    .body("Kan ikke opprette for ident " + ident + "fordi personen allerede finnes AAREG (" + miljo + ").");
```

Comment:
`.body("Kan ikke opprette arbeidsforhold for ident " + ident + " fordi personen allerede finnes AAREG (" + miljo + ").");`